common/buildings/00_duchy_capital_buildings.txt

```diff
@@ -341,6 +341,9 @@ siege_works_02 = {
 	character_modifier = {
```

Comment:
You should relocalize these modifiers since they give a bonus to new unit types.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104640065-94335100-56c1-11eb-938b-e0d34d0b2a1e.png)

</details>